chance/p5js/chance-praegnanz/index.md

```diff
@@ -3,9 +3,9 @@ layout: p5js
 author: josues
 author-url: https://incom.org/profil/7029
 title: Law of praegnanz and chance
-tags: P5.js, chance, proximity, Law of praegnanz
+tags: [P5.js, chance, proximity, Law of praegnanz]
 permalink: /chance/chance-praegnanz/
 summary: This creates random Praegnanzen between ellipses.
 ---  
 
-![](https://raw.githubusercontent.com/josues/gestalten-in-code/master/chance/p5js/chance-praegnanz/out.png)  
+![](./out.png)   
```

Comment:
Thanks. Having a relativ path is better. But the code from the index.js will be added automagically to the site. So there is no need to repeat the image below the actual canvas.